<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="X390" ignored="0" name="Test" tests="3" failures="1" timestamp="2020-12-09T21:56:16 EST" time="11.886" errors="0">
  <testcase name="logoPresence" time="0.089" classname="allureReport.aller.one"/>
  <testcase name="loginTest" time="3.829" classname="allureReport.aller.one">
    <failure type="java.lang.AssertionError" message="expected [nopCommerce demo store123] but found [nopCommerce demo store. Login]">
      <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store123] but found [nopCommerce demo store. Login]
at allureReport.aller.one.loginTest(one.java:66)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="registrationTest" time="0.003" classname="allureReport.aller.one">
    <skipped/>
  </testcase> <!-- registrationTest -->
</testsuite> <!-- Test -->
